backend/director/db/postgres/db.py

```
@@ -0,0 +1,236 @@
+        )
+        self.cursor = self.conn.cursor(cursor_factory=RealDictCursor)
+        logger.info("Connected to PostgreSQL DB..........")
+
+    def create_session(
+        self,
+        session_id: str,
+        video_id: str,
+        collection_id: str,
+        created_at: int = None,
+        updated_at: int = None,
+        metadata: dict = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)